desicochrane/basic-mailbox-api
# Add mailbox to domain

**🤗commit-by-commit-friendly🤗**

# Why does this PR exist?

For the messages api, each message has a sender. We can represent this with a "mailbox" entity where a mailbox can send a message (and possibly later could receive a message too).

Because a message will require a sending mailbox, we should add the mailbox to our domain first.

### How should this be manually tested?

1. run `php artisan migrate` to create the table and seed your admin user
1. check out the tests `phpunit`

### Discussion

Moving forward it could make sense to link a mailbox to a user, but as a user system was not a requirement for this prototype I have not included it yet.
